cfep-21.md

```diff
@@ -0,0 +1,60 @@
+
+<table>
+<tr><td> Title </td><td> Conda Forge-Bioconda Transfer Policy </td>
+<tr><td> Status </td><td> Draft </td></tr>
+<tr><td> Author(s) </td><td> Mervin Fansler &lt;mervin.fansler@bric.ku.dk&gt;</td></tr>
+<tr><td> Created </td><td> 10 July 2024</td></tr>
+<tr><td> Updated </td><td> 18 July 2024</td></tr>
+<tr><td> Discussion </td><td> https://github.com/conda-forge/cfep/pull/52 </td></tr>
+<tr><td> Implementation </td><td> NA </td></tr>
+</table>
+
+## Abstract
+
+Bioconda (BC) and Conda Forge (CF) have occasionally transferred package recipes between them. The `strict` channel
+priority recommended to BC users and used when building packages on the BC channel poses an
+issue when transfers occur, especially in the CF-to-BC direction. This proposal aims to 
+define a policy for standard procedure when such transfers occur. Included in this is the introduction
+of a new label to be applied to all package builds that transfer CF-to-BC.
+
+## Specification
+
+### Bioconda to Conda Forge (BC-to-CF) Transfers
+
+1. Packages on BC considered for transfer should be marked with a ["Move to Conda Forge" label](https://github.com/bioconda/bioconda-recipes/labels/Move%20to%20Conda-Forge) in the `bioconda-recipes` repository. This label can be applied either in an Issue or a Pull Request.
+2. A Pull Request should be create on CF's `staged-recipes` repository to add the
+3. The linter will flag when adding a recipe already hosted by BC. This will trigger a message to [the **conda-forge/bioconda-recipes** team](https://github.com/orgs/conda-forge/teams/bioconda-recipes).
+4. Upon approval by a **conda-forge/bioconda-recipes** team member, the Pull Request can then be reviewed and merged.
```

Comment:
I guess we should get sign off from bioconda members in this CFEP too. cc @conda-forge/bioconda-recipes

Reply:
Are you thinking a specific number? @bgruening and I are both members.